Assets/Scripts/PlayerControls.cs

```diff
@@ -9,18 +9,28 @@ public class PlayerControls : MonoBehaviour
 	[SerializeField] float _interactCheckRadius = 0.2f;
 
 	PlayerActor _actor;
-	ActorPhysics _actorPhysics;
-	Cutting _cutting;
 	
 	Button _holdButton = new Button( "Hold" );
+	public Button holdButton
+	{
+		get { return _holdButton; }
+	}
```

Comment:
The button is also for gliding, so  I don't feel like those work any better. It makes sense if you think of the two inputs as "use" and "hold", though I get why that looks weird at first glance. I'm up to changing it if you have something that describes it better.